sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1107,6 +1107,14 @@ object functions {
    */
   def input_file_name(): Column = withExpr { InputFileName() }
 
+  /**
+   * Return true iff the column is Infinity.
+   *
+   * @group normal_funcs
+   * @since 2.4.0
+   */
+  def isinf(e: Column): Column = withExpr { IsInf(e.expr) }
```

Comment:
Mind if I ask to elaborate  `isinf` vs `isInf` across the APIs?

Reply:
I have followed what seemed to be the preexistent convention for function names in those particular files. For example in `functions.scala` all  function names were in lower case but in `Column.scala` all function names were in camel case.

Reply:
I guess someone should elaborate if Column.isFoo vs function's isfoo is the right pattern we want to stay with...